### DISCUSSION TITLE
How do you set up this tool to check alt text?

### DISCUSSION THREAD

**Discussion by freenandes** [davidlday/vscode-languagetool-linter#643] [Q&A]
I'm using this extension to check in markdown files. How do i get it to check inside the alt area, like inside the `![this part of the link](url.to/image.jpg)`. I guess this applies to link naming also since it's the same syntax. How?

**Comment by davidlday** [comment]
I'll have to look into this one in the underlying parser, and will open an issue on the appropriate repo (might not be this one). I can see it's getting skipped as well.
